tests/maintel/test_make_lsstcam_calibrations.py

```diff
@@ -0,0 +1,120 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+
+import logging
+import unittest
+
+import pytest
+from lsst.ts import externalscripts, salobj, standardscripts
+from lsst.ts.externalscripts.maintel import MakeLSSTCamCalibrations
+
+logger = logging.getLogger(__name__)
+logger.propagate = True
+
+
+class TestMakeLSSTCamCalibrations(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    async def basic_make_script(self, index):
+        logger.debug("Starting basic_make_script")
+        self.script = MakeLSSTCamCalibrations(index=index)
+
```

Comment:
To prevent the Script from creating the camera and ocsp objects, make sure you add:

```
self.script._lsstcam = AsyncMock()
self.script._ocps_group = AsyncMock()
```

here. You will have to `from unittest.mock import AsyncMock` at the beginning.